query/control/controller.go

```diff
@@ -121,6 +128,9 @@ func (c *Config) validate(isComplete bool) error {
 	if c.ConcurrencyQuota <= 0 {
 		return errors.New("ConcurrencyQuota must be positive")
 	}
+	if c.ConcurrencyQuota > MaxQueryConcurrency {
```

Comment:
Should we define ConcurrencyQuota as an int64 in the struct? I realize that `int` is 64 bits on our target architectures, but the additional clarity couldn't hurt.

Reply:
`int32` is what we'd want/need to use, but I'm a fan of the general idea... it might get us some extra validation "for free" too, if the CLI parser is smart about detecting overflows. I'll give it a try in the morning.

Reply:
Using `int32` does give us some validation at the CLI layer. I'll keep heading down this path.